modules/test-support/src/main/scala/dev/profunktor/redis4cats/DockerRedis.scala

```diff
@@ -139,7 +139,7 @@ object DockerRedis {
       s"Redis $image startup observer"
     )
     thread.start()
-    val id = dockerId.get
+    val id = dockerId.poll()
```

Comment:
Are you sure you want `poll` here as opposed to `peek`? You're changing the semantics of here since `poll` removes the element as opposed to just retrieving it. I don't know if that's okay given context, but it's different than what was here before.

Reply:
Thanks for pointing that out. I didn't know it changed the semantics as I don't use this, I just run a docker instance of Redis to test (works faster) but will be reviewed before a next release, now's a bit late unfortunately.